library/src/hspec/mingw/mal_hspec_mingw_adc.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * mal_hspec_mingw_adc.h
+ *
+ *  Created on: Mar 24, 2016
+ *      Author: Olivier
+ */
+
+#ifndef HSPEC_MINGW_MAL_HSPEC_MINGW_ADC_H_
+#define HSPEC_MINGW_MAL_HSPEC_MINGW_ADC_H_
+
+#include "std/mal_error.h"
+#include "hspec/mal_hspec_cmn.h"
+#include "std/mal_stdint.h"
+
+mal_error_e mal_hspec_mingw_get_valid_adc_ios(mal_hspec_adc_e adc,const mal_hspec_gpio_s **ios, uint8_t *size);
+
+void mal_hspec_mingw_add_adc(mal_hspec_adc_e adc);
```

Comment:
True, this is obsolete. I changed my mind half way threw coding this. It is normal, it is not used, I will remove this definition.